src/gigs-board/entity/community/configurator.jsx

```diff
@@ -284,25 +286,26 @@ const CommunityAboutSchema = {
   },
 
   twitter_handle: {
-    inputProps: { min: 2, max: 60 },
-    label: "Twitter handle",
+    inputProps: { prefix: "https://twitter.com/", min: 2, max: 60 },
+    label: "Twitter",
     order: 2,
   },
 
   github_handle: {
-    inputProps: { min: 2, max: 60 },
-    label: "Github organization handle",
+    inputProps: { prefix: "https://github.com/", min: 2, max: 60 },
+    label: "Github",
     order: 3,
   },
 
   telegram_handle: {
+    inputProps: { prefix: "https://t.me/", min: 2, max: 60 },
     format: "comma-separated",
-    label: "Telegram handles",
+    label: "Telegram",
```

Comment:
telegram is now a comma separated string, allows a community to have multiple telegram handles. For example: if this field is set to `nearnft,nearprotocol`, telegram page will render messages from https://t.me/nearnft` and `https://t.me/nearprotocol`. This change was happening after #173 created, so the prefix for telegram isn't the best way, maybe just keep that one simple and indicate user to fill comma separated handles in placeholder?